version/K8S.VERSION

```diff
@@ -1 +1 @@
-v1.3.0-1.15.x
+v1.5.0-1.15.x
```

Comment:
no need to change this

Reply:
will this change cause any bug?

Reply:
> will this change cause any bug?

I think no but why do you change it ?

Reply:
Because this refer to cluster v1.5.0, however I think this file is deprecated and do not needed any more?

Reply:
Suggest to leave this file unchanged, we can handle this issue in v1.6:  https://github.com/microsoft/pai/issues/5132

Reply:
ok